src/view/actions/sendEvent.jsx

```diff
@@ -233,6 +239,31 @@ const sendDisplayEventUnchecked = disabledCheckbox({
   beta: true
 });
 
+const defaultPersonalizationEnabledField = radioGroup({
+  name: "defaultPersonalizationEnabled",
+  label: "Request default personalization",
+  dataElementSupported: false,
+  defaultValue: "auto",
+  items: [
+    {
+      value: "auto",
+      label:
+        "Automatic - request default personalization when it has not yet been requested."
+    },
+    {
+      value: "true",
+      label:
+        "Enabled - explicitly request the page scope and default surface. This will also refresh the view cache."
```

Comment:
We have talked about the SPA cache and it's behavior when new propositions are returned from server for an SPA view and we agreed to update the SPA cache instead of refreshing it. Did something changed?

Reply:
`view cache` => `SPA view cache`

Reply:
Yes you are right, I changed the cache to update. I guess I was thinking of refresh and update as synonyms. I will change this to "update".